packages/hmr-plugin/src/internal/do-bootstrap.ts

```diff
@@ -0,0 +1,22 @@
+import { NgModuleRef } from '@angular/core';
+import { StateContext } from '@ngxs/store';
+
+import { setStateInHmrStorage } from './common';
+import { stateEventLoop } from './state-event-loop';
+import { HmrNgxsStoreOnInit, NgxsHmrLifeCycle, NgxsStoreSnapshot } from '../symbols';
+
+export function hmrDoBootstrap<T extends NgxsHmrLifeCycle<S>, S = NgxsStoreSnapshot>(
+  ref: NgModuleRef<T>
+): NgModuleRef<T> {
+  const ngxsHmrLifeCycle: NgxsHmrLifeCycle<S> = ref.instance;
```

Comment:
The same as the comment in `on-destroy.ts`.
Also, the typings should be removed/included in the other file, to keep consistency.